agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -542,6 +542,9 @@ func (dg *dockerGoClient) CreateContainer(ctx context.Context,
 	timeout time.Duration) DockerContainerMetadata {
 	ctx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
+	// Uncomment the following line to run tasks with FireLens on Windows
```

Comment:
I tried this for a little while, but ran out of time to trace through everything. The locations where I tried it had the user variable reset back to 0 by the time the container started with this method. It will require more work to find where and when to set the User. This is just the workaround I used to run tasks in my demo during the presentation.